src/GraphControl/Control/Grapher.cpp

```diff
@@ -199,6 +199,8 @@ namespace GraphControl
         {
             if (auto analyzer = graph->GetAnalyzer())
             {
+                graph->GetOptions().SetAllowKeyGraphFeaturesForFunctionsWithParameters(true);
+
```

Comment:
Not related to this PR, but since this is creating a new graph, are we not passing along the units (Radians, degrees, gradians)? Would this alter the output of the analysis?

Reply:
The analysis updates according to the trig units.

Reply:
Is that correct? If the graph is set to degrees shouldn't the analysis also be in degrees?

Reply:
nvm, as we discussed this is done in the MathSolver not in the Graph!